README.md

```diff
@@ -1,79 +1,69 @@
 # CSI NFS driver
 
-## Kubernetes
-### Requirements
+## Overview
 
-The folllowing feature gates and runtime config have to be enabled to deploy the driver
+This is a repository for [NFS](https://en.wikipedia.org/wiki/Network_File_System) [CSI](https://kubernetes-csi.github.io/docs/) Driver.
+Currently it implements bare minimum of the CSI and is in the alpha state of the development.
 
-```
-FEATURE_GATES=CSIPersistentVolume=true,MountPropagation=true
-RUNTIME_CONFIG="storage.k8s.io/v1alpha1=true"
-```
+#### CSI Feature matrix
 
-Mountprogpation requries support for privileged containers. So, make sure privileged containers are enabled in the cluster.
+| **nfs.csi.k8s.io** | Min K8s version | CSI versions compatibility | Dynamic Provisioning | Resize | Snapshots | Raw Block | AccessModes              | Status |
+|--------------------|-----------------|----------------------------|----------------------|--------|-----------|-----------|--------------------------|--------|
+|master              | 1.14            | v1.1                       |  no                  |  no    |  no       |  no       | Read/Write Multiple Pods | Alpha  |
+|v1.0.0              | 1.14            | v1.1                       |  no                  |  no    |  no       |  no       | Read/Write Multiple Pods | Alpha  |
```

Comment:
Just a tiny nit, let's start the versioning at 2.0 to distinguish it from the old driver.

Reply:
sure, just so I understand what you mean, CSI version to bump to v2.0?

Reply:
maybe I can clarify the table a bit too to avoid any confusion, `master` == `v1.0.0`, there has not been a release of this yet. Theoretically we can start with `v2.0.0` if that is what you were referring to, but that means `v1.0.0` will probably never be released :)

Reply:
The problem is we actually do have a 1.0 released, in the old repo: https://github.com/kubernetes-csi/drivers/releases, and we're going to reuse the the same image repo, so 1.0 is already taken.

Reply:
ah, now with the second repo it makes a lot of sense. My bad, adjusted the README and manifest + version in code to reflect the desired v2.0.0 release version. Thanks for clarifying that